lib/build-view.js

```diff
@@ -95,6 +99,14 @@ module.exports = (function() {
     this.detach(true);
   };
 
+  BuildView.prototype.toggle = function(event, element) {
+    if (this.isAttached()) {
+      this.detach(true);
+    } else {
+      this.attach();
```

Comment:
force flag has been added to attach as well in commit cc66db0 this must be passed here as well.

Also, I think this can be written neater by

    this.isAttached() ? this.detach(true) : this.attach(true);